modules/cluster-logging-deploy-cli.adoc

```diff
@@ -308,35 +315,42 @@ spec:
   managementState: "Managed"  <2>
   logStore:
     type: "elasticsearch"  <3>
+    retentionPolicy: <4>
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
     elasticsearch:
-      nodeCount: 3 <4>
+      nodeCount: 3 <5>
       storage:
-        storageClassName: "<storage-class-name>" <5>
+        storageClassName: "<storage-class-name>" <6>
         size: 200G
       redundancyPolicy: "SingleRedundancy"
   visualization:
-    type: "kibana"  <6>
+    type: "kibana"  <7>
     kibana:
       replicas: 1
   curation:
-    type: "curator"  <7>
+    type: "curator"  <8>
     curator:
       schedule: "30 3 * * *"
```

Comment:
Curator will be dropped in the future, so I think we'd better remove this filed in 4.5 DOC.

Reply:
It's probably better to add a note this is deprecated and will be removed in future.  Curator settings are still valid but the expectation is they are only used to remove data from the old index format.  "Curation" going forward is by using the new datamodel and rolling indices

Reply:
@QiaolingTang @jcantrill @sichvoge 
I left the Curator setting in the installing topics, where you create the cluster logging CR with new wording. I removed all other sections on configuring curator.
_Settings for configuring the Curator schedule. Curator is used to remove data that is in the Elasticsearch index format prior to {product-title} 4.5 and will be removed in a later release._